packages/core/src/client/index.ts

```diff
@@ -9,6 +9,7 @@ import { proxySet } from "valtio/utils";
 import { createPromiseWithTimeout } from "../utils/promiseTimeout";
 import { isSmartWallet } from "../utils/address";
 import { version as web3inboxCorePackageVersion, name as web3inboxCorePackageName } from '../../package.json'
+import { debounce } from 'lodash'
```

Comment:
It's essentially in most JS packages - you can almost assume it's a sub dependency of your project. Since we have the debounce function already in our package I don't see a reason of writing it again